### PR TITLE
Update readme to remove unicode characters which break pip install

### LINKED ISSUE #66: install on windows 10 gbk error
I install it on windows ,but it shows "UnicodeDecodeError: 'gbk' codec can't decode byte 0x94 in position 1295: illegal multibyte sequence" error.

### PR DESCRIPTION
**Pull request by bmcmurray**
This should fix #66 which is also occurring for me on OS X High Sierra with Python 3.6.

The issue is that `setup.py` loads the `README.rst` as the `long_description` and the README contains unicode characters that pip can't handle while installing.

### CODE REVIEW

**Comment by yakky** [comment]
@bmcmurray thanks for sharing this solution
Unfortunately the encoded characters are usually not decoded correctly.
To me we can replace mark and "x" characters with ascii equivalent (say "x" and "-" respecively)
But calling in @FinalAngel for input

**Comment by FinalAngel** [comment]
I'd rather say that there was a hidden invalid character, the encoding there shouldn't throw an error but you never know... systems :)

**Comment by yakky** [comment]
I edited my comment because I think my point wasn't clear

TL;DR: how about replacing unicode chars with ascii ones?

It's a weird issue but I guess that playing on the safe side it would be a good option here

**Comment by yakky** [comment]
Fixed in #74 which removed the table completely
